app/dbConnection/index.js

```diff
@@ -63,6 +63,29 @@ exports.getWebsitesToEntities = function () {
     });
   });
 };
+
+exports.getPromisingWebsites = function () {
+  return new Promise(resolve => {
+    connect().then(() => {
+      let sortedWebsites = [];
+
+      console.log('Query the db');
+      // TODO: remove the limit
+      context.sequelize.query(
+        'SELECT * from (SELECT blob_url, COUNT(*) as count FROM (select * from websites limit 100) w LEFT JOIN contains c ON w.id = c."websiteId" GROUP BY blob_url ORDER BY count DESC) a where a.count > 50;',
```

Comment:
raw SQL queries are the best queries

should be fine without string interpolation, though